pom.xml

```diff
@@ -1553,7 +1553,7 @@ under the License.
 							random: enable it randomly, unless explicitly set
 							unset: don't alter the configuration
 						-->
-						<checkpointing.changelog>random</checkpointing.changelog>
+						<checkpointing.changelog>unset</checkpointing.changelog>
```

Comment:
you could add a reference to the FLINK-25399 ticket in the comment why is this disabled.